README.md

```diff
@@ -263,5 +263,26 @@ EXAMPLE
   $ spot validate api.ts
 ```
 
+_See code: [build/cli/src/commands/validate.js](https://github.com/airtasker/spot/blob/v0.2.13/build/cli/src/commands/validate.js)_
+
+## `spot tag SPOT_CONTRACT`
+
+Generate a version tag based on a Spot contract
+
+```
+USAGE
+  $ spot tag SPOT_CONTRACT
+
+ARGUMENTS
+  SPOT_CONTRACT  path to Spot contract
+
+OPTIONS
+  -h, --help             show CLI help
+  --algorithm=algorithm  digest algorithm used to generate the hash
```

Comment:
I'd prefer not to be able to specify an algorithm. It's important that two different people running `spot tag` on the same contract (with the same version of Spot) end up with the same result.

Allowing you to pick a specific algorithm introduces uncertainty: if you see a different result, is it because the other person had used a different algorithm?

Based on this, I think we should hardcode the choice of algorithm.

Reply:
yea i actuallt hesitated. Makes sense to hardcode this.

Reply:
Reminder to remove it from the README now that you've removed the code. Feel free to resolve this comment then.